docusaurus/docs/iOS/03-guides/10-view-slots.mdx

```diff
@@ -62,13 +62,11 @@ The video participants view slot presents the grid of users that are in the call
 public func makeVideoParticipantsView(
     participants: [CallParticipant],
     availableSize: CGSize,
-    onViewRendering: @escaping (VideoRenderer, CallParticipant) -> Void,
```

Comment:
We don't use this in the tutorials, only in the docs?

Reply:
It seems that the tutorial not include this.